src/Sarif.Converters/FortifyFprConverter.cs

```diff
@@ -409,25 +433,7 @@ private void ParseVulnerability()
                 {
                     // Get the rule GUID from the ClassId element.
                     string ruleId = _reader.ReadElementContentAsString();
```

Comment:
>ruleId [](start = 27, length = 6)

The guid isn't really appropriate as a rule id. Furthermore, I'm not sure we're setting this value on the appropriate reportingDescriptor.guid property (where it belongs)? Not your problem, maybe we can file an issue?

Reply:
It looks like Chris had a discussion with Yekaterina about this and ended up committing this way. See https://github.com/Microsoft/sarif-sdk/pull/1394. Do you want an issue in the sarif-sdk to investigate further?